tdrs-frontend/package.json

```diff
@@ -74,11 +74,11 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-prettier": "^4.0.0",
+    "faker": "^5.5.3",
     "jest-enzyme": "^7.1.2",
-    "node-sass": "^5.0.0",
+    "node-sass": "^6.0.1",
```

Comment:
Two notable things here:

1. Major versions of `node-sass` are incompatible across major Node versions. Thus we upgrade `node-sass`.
2. Likewise, many transitive dependencies will want to resolve different versions based on the version of Node they are being installed with, thus the giant `yarn.lock` file below.

(note that `faker` is just being sorted automatically in the list)